### ISSUE TITLE
New logic for comparing locations

### ISSUE THREAD

**Issue by diegoliberman** [instedd/planwise#556]
Cambiar la función de comparación de nuevas locaciones para considerar una mezcla balanceada del impacto en la demanda y la cobertura geográfica

**Comment by diegoliberman** [comment]
Entiendo que esto impacta en las nuevas facilities que el sistema sugiere construir/ampliar.

**Comment by ggiraldez** [comment]
Correct. This applies to the suggestion algorithm. Currently it only considers total potential demand satisfaction, but it should also weigh in the geographic coverage. We may also want to apply this strategy when comparing the different scenarios, especially for sorting and labeling them (ie. "best").
